ctk/common/src/main/kotlin/org/codice/compliance/verification/profile/SingleSignOnProfileVerifier.kt

```diff
@@ -57,6 +58,7 @@ class SingleSignOnProfileVerifier(private val authnRequest: AuthnRequest,
     fun verify() {
         verifyIssuer()
         verifySsoAssertions()
+        SubjectComparisonVerifier(authnRequest, samlResponseDom).verifySubjectsMatchSSO()
```

Comment:
❓ Shouldn't you pass in `null` cause we're comparing the Response's Subjects? If not we won't need the `request: Node? = null` in the `SubjectComparisonVerifier`'s constructor